vhd/packer/configure-windows-vhd.ps1

```diff
@@ -85,13 +85,11 @@ function Get-FilesToCacheOnVHD
         "c:\akse-cache\win-k8s\" = @(
             "https://acs-mirror.azureedge.net/wink8s/azs-v1.14.7-1int.zip",
             "https://acs-mirror.azureedge.net/wink8s/azs-v1.14.8-1int.zip",
-            "https://acs-mirror.azureedge.net/wink8s/azs-v1.15.7-1int.zip",
-            "https://acs-mirror.azureedge.net/wink8s/azs-v1.15.9-1int.zip",

```

Comment:
Should we add `https://kubernetesartifacts.azureedge.net/kubernetes/v1.15.12-azs/windowszip/v1.15.12-azs-1int.zip` and friends to the Windows VHD? That seems to be the current URL where this archive is published.

Reply:
Yes, I have a PR ready, just waiting for v1.15.10

Reply:
Will free to update if you like

Reply:
Let's do those changes in a separate PR just to keep it simple.